content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,417 @@
+---
+title: Pod Eviction
+content_type: concept
+weight: 50
+---
+
+Eviction is the process by which {{<glossary_tooltip text="pods" term_id="pod" length="short">}}
+in a {{<glossary_tooltip text="node" term_id="node" length="short">}} are
+terminated. 
+
+In Kubernetes, there are two types of pod eviction:
+
+  1. Graceful eviction
+  2. Out-of-resource eviction ({{<glossary_tooltip text="kubelet" term_id="kubelet" length="short">}}
+     eviction)
+
+{{<caution>}}
+Out-of-resource eviction does not share any characteristics
+with graceful eviction. Do not use the two eviction types interchangeably. 
+{{</caution>}}
+
+## Graceful eviction
+
+{{<glossary_definition term_id="graceful-eviction" length="all">}}
+
+Graceful evictions respect your configured [`PodDisruptionBudgets`](/docs/tasks/run-application/configure-pdb/)
+and [`terminationGracePeriodSeconds`](/docs/concepts/workloads/pods/pod-lifecycle#pod-termination). 
+
+## Out-of-resource eviction
+
+{{<glossary_definition term_id="kubelet-eviction" length="all">}}
+
+During an out-of-resource eviction, the kubelet sets the `PodPhase` for the
+selected pods to `Failed`. This terminates the pods. 
+
+The kubelet does not respect your configured `PodDisruptionBudget` or the pod's
+`terminationGracePeriodSeconds`, and uses a `0s` grace period for termination.
+
+If the pods are managed by a {{<glossary_tooltip text="workload" term_id="workload" length="short">}}
```

Comment:
```suggestion
If the pods are managed by a {{<glossary_tooltip text="workload" term_id="workload">}}
```
(`length` isn't relevant to tooltips; these either use the default text or the text you explicitly use as an override)